tree-items.txt

```diff
@@ -506,14 +506,17 @@ xfs/ext4). In BTRFS a file extent can be in one of 3 states:
 
 * BTRFS_FILE_EXTENT_INLINE - an extent item who can hold all of the actual file
 data in its body is considered an inline extent. In such cases data is read
-directly from the body of the extent.
+directly from the body of the extent. The current default value is 2048 bytes,
+but it can be changed with max_inline mount option. The max_inline value is then
+calculated with min(max_inline_value, sectorsize).
 
 * BTRFS_FILE_EXTENT_PREALLOC - a pre-allocated extent. These are ranges which
-have been reserved by using fallocate.
+have been reserved by using fallocate. Each extent can map up to 256Mb for
+pre-allocated extents.
```

Comment:
That's correct, however I'm not entirely sure that level of detail is really relevant? And also can be changed at any time.  What's your motivation for bringing this up ?

Reply:
Following discussion on irc I'm ok with this change.